python_examples/intervention_example/intervention_simulation_flow.py

```diff
@@ -0,0 +1,83 @@
+#
```

Comment:
Do we need this as well as the other intervention simulation script?

Reply:
Yes, I think it is nice to show how all interventions can be called in the same simulation. We should make a nice example when all interventions are implemented.

Reply:
If this has all the interventions (case isolation, place closure, household quarantine) then I think we probably don't need separate workflows for all of those as well? Let me know what you think though - we talked about adding a README to make it clearer what each file was for so maybe that will help!

Reply:
I think separate workflows for each intervention is still needed. It helps to clearly see how this single intervention effects the overall performance (only change one parameter value). When implementing all interventions at the same time, it's hard to differ the single intervention effect.

Reply:
We discussed this and decided that it is not needed. I removed the script on the 'improve_interventions'  branch which is now merged.